realm/realm-library/src/main/cpp/io_realm_SyncManager.cpp

```diff
@@ -59,3 +62,13 @@ Java_io_realm_SyncManager_nativeRunClient(JNIEnv *env, jclass)
         sync_client->run();
     } CATCH_STD()
 }
+
+JNIEXPORT void JNICALL
+Java_io_realm_SyncManager_nativeConfigureMetaDataSystem(JNIEnv *env, jclass,
+                                                        jstring baseFile) {
+    TR_ENTER()
+    try {
+        JStringAccessor base_file_path(env, baseFile); // throws
+        SyncManager::shared().configure_file_system(base_file_path, SyncManager::MetadataMode::NoEncryption);
```

Comment:
no, this is what motivated the creation of https://github.com/realm/realm-android-user-store 

on iOS they can obtain a key directly from the keychain, which is not the case for Android, since we need the KeyStore to be unlocked & does involve user interaction & lifecycle